From 4036d3042200a6d796026db4ef7db8153a37f39e Mon Sep 17 00:00:00 2001
From: Carsten Haitzler (Raster <raster@rasterman.com>
Date: Thu, 30 Oct 2008 17:09:02 +1100
Subject: [PATCH] palmt650.conf: use palmt650-keyboard-modmap in palmt650 machine conf

use the new palmt650-keyboard-modmap package in the palmt650 config so the
keymap is right for the treo 650.
---
 conf/machine/palmt650.conf |    5 ++++-
 1 files changed, 4 insertions(+), 1 deletions(-)

diff --git a/conf/machine/palmt650.conf b/conf/machine/palmt650.conf
index 272c0dc..f81ade3 100644
--- a/conf/machine/palmt650.conf
+++ b/conf/machine/palmt650.conf
@@ -12,6 +12,9 @@ GUI_MACHINE_CLASS = "bigscreen"
 
 MACHINE_EXTRA_RRECOMMENDS += "kernel-module-snd-pxa2xx-ac97 \
                            kernel-module-pxa27x-keyboard \
-                           kernel-module-wm97xx-ts"
+                           kernel-module-wm97xx-ts \
+			   "
+
+MACHINE_EXTRA_RDEPENDS += "palmt650-keyboard-modmap"
 
 module_autoload_g_ether = "g_ether"
-- 
1.5.4.3

